### PR TITLE
[PRD-2375] call hwclock --systohc during provisioning

### PR DESCRIPTION
**Pull request by mattymo**

### CODE REVIEW

File: deployment/puppet/cobbler/templates/snippets/ntp_to_masternode_ubuntu.erb [hunk 1]
```diff
@@ -3,3 +3,5 @@ sed -i '1 i tinker panic 0' /target/etc/ntp.conf && \
 echo 0 > /target/var/lib/ntp/drift && \
 sed -i '/^\s*server/ d' /target/etc/ntp.conf && \
 echo "server $server burst iburst" >> /target/etc/ntp.conf && \
+ntpdate -t 4 -b $server
```
Comment (xdel) [inline deployment/puppet/cobbler/templates/snippets/ntp_to_masternode_ubuntu.erb]: Nope, line iterators are absolutely necessary for d-i

File: deployment/puppet/cobbler/templates/snippets/ntp_to_masternode_ubuntu.erb [hunk 2]
```diff
@@ -3,3 +3,5 @@ sed -i '1 i tinker panic 0' /target/etc/ntp.conf && \
...
+hwclock --systohc
```
Comment (xdel) [inline deployment/puppet/cobbler/templates/snippets/ntp_to_masternode_ubuntu.erb]: Please check at least once if it works in the preseed.

**Comment by xdel** [comment]
+1

**Comment by aglarendil** [comment]
+2

**Pull request merged**